src/index.js

```diff
@@ -0,0 +1,22 @@
+import './style.css';
```

Comment:
Even when your style looks good it doesn't look like the one of the video. Please try adding the corresponding changes to look like the one on the video. It seems that your CSS is not being loaded by Webpack Style/CSS Loader.

Check the image for better understanding:
![Capture](https://user-images.githubusercontent.com/75709904/149544724-510a3552-9ac6-4839-ae2d-d73c224f8ff5.PNG)